storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -720,6 +720,15 @@ Options
 
    RocksDB WAL directory.
 
+.. option:: --rocksdb-checkpoint-max-age
+
+   The checkpoint cannot be older than this number of seconds when the backup
+   completes.
+
+.. option:: --rocksdb-checkpoint-max-count
+
+   Complete the backup even if the checkpoint age requirement has not been met after
+   this number of checkpoints.
 
```

Comment:
`–-backup-lock-timeout` and `-–backup-lock-retry-count` should be mentioned

Reply:
Need more information about these options

Reply:
Added based on PXB-1914